tests/test_chrono.py

```diff
@@ -0,0 +1,114 @@
+
+
+def test_chrono_system_clock():
+    from pybind11_tests import test_chrono1
+    import datetime
+
+    # Get the time from both c++ and datetime
+    date1 = test_chrono1()
+    date2 = datetime.datetime.today()
+
+    # The returned value should be a datetime
+    assert isinstance(date1, datetime.datetime)
+
+    # The numbers should vary by a very small amount (time it took to execute)
+    diff = abs(date1 - date2)
+
+    # There should never be a days/seconds difference
+    assert diff.days == 0
+    assert diff.seconds == 0
+
+    # 50 milliseconds is a very long time to execute this
+    assert diff.microseconds < 50000
```

Comment:
Although that's true, this is quite a useful test (i.e. are two datetimes from python and C++ giving the same date and time, as expected?).  The '< 50000' could probably be dropped, however (keeping the `diff.days`/`.seconds` == 0 checks) to relax this to second precision.  For that to fail, we'd have to have the system clock change by a full second between the execution of lines 8 and 9: and if it's going that slow, there are serious problems.